docs/contributors/code/react-native/osx-setup-guide.md

```diff
@@ -71,28 +40,46 @@ npm run distclean
 npm ci
 ```
 
-## Unit Tests
+## iOS
 
-Unit tests should work at this point.
+### CocoaPods
 
-```sh
-npm run native test
+[CocoaPods](https://guides.cocoapods.org/using/getting-started.html) is needed to fetch React and third-party dependencies:
+
+```
+sudo gem install cocoapods
```

Comment:
I wonder if it makes sense to add some mention that this assumes using the system Ruby (hence the need for `sudo`). Additionally, though I'm not sure this has to be done in this PR, perhaps we can start recommending the Ruby version manager we mainly use (`rbenv`) as part of the setup. I think in that case, one doesn't need to manually install the `cocoapods` gem. Thoughts?

Reply:
That's a good idea! I've gone ahead to make an update in https://github.com/WordPress/gutenberg/pull/40964/commits/421a1a802c6906ac26bed785e74dfba7ca2e3c8b. I kept the mention of how to install the gem in the case that someone is only using the system default. My thought behind that is that it may be simpler if the person only wants to use the demo app (and doesn't need to set up the native app). What do you think?

Reply:
makes sense to me 👍